html/semantics/embedded-content/the-img-element/image-loading-lazy-slow-aspect-ratio.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html style="reftest-wait">
+<html class="reftest-wait">
```

Comment:
@chrishtr I think this might be (part of) the real reason why this was flaky... (I've pushed a fix for you. It'll be merged if it's no longer flaky and then imported back to Chromium.)